client/knowledge_base_search/src/containers/TrainingContainer/TrainingComparison.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { string, number, shape } from 'prop-types';
+import ResultContainer from '../ResultContainer/ResultContainer';
+import replaceNewlines from '../../utils/replaceNewlines';
+import './styles.css';
+
+class TrainingComparison extends Component {
+  render() {
+    const { regularResult, trainedResult, index } = this.props;
+
+    return (
+      <div className="training_comparison">
+        <div className="training_comparison--content">
+          <div className="training_comparison--content_left">
+            { index === 0 && (<h5>Standard search</h5>) }
```

Comment:
Maybe break this off into a `const isFirst = index === 0` or maybe even perform the check where the iteration is happening and pass it down as a prop.

Reply:
done